tests/dummy/app/pods/application/template.hbs

```diff
@@ -1 +1,3 @@
-{{outlet}}
+<div class="dummy-body">
```

Comment:
@sglanzer The documentation site actually imports the application.hbs , replacing the outlet with the demo.hbs